games-fps/prboom-plus/prboom_plus-2.5.1.4.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="Enhanced clone of the classic first-person shooter Doom"
+DESCRIPTION="PrBoom+ is a Doom source port developed from the original PrBoom
+ project by Andrey Budko"
+HOMEPAGE="http://prboom-plus.sourceforge.net"
+COPYRIGHT="2016 Andrey Budko et al."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/prboom-plus/prboom-plus-$portVersion.tar.gz"
+CHECKSUM_SHA256="e0e2241d174839b107f1a42b191aa9895dc6749e477cbc850ad520fba2353b94"
+SOURCE_DIR="prboom-plus-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+
+commandBinDir="$binDir"
+if [ "$targetArchitecture" = "x86_gcc2" ]; then
+	SECONDARY_ARCHITECTURES="x86"
+	commandBinDir="$prefix/bin"
+fi
+
+PROVIDES="
+	cmd:prboom_plus = $portVersion
```

Comment:
```diff
 PROVIDES="
+	prboom_plus$secondaryArchSuffix = $portVersion
 	cmd:prboom_plus = $portVersion
```

We also need an entry for the package name, otherwise it wouldn't install with `HaikuDepot`